backend/app/api/endpoints/openapi_responses.py

```
@@ -0,0 +1,550 @@
+        source="api",
+        model_id=model_info.get("model_id"),
+        force_override_bot_model=model_info.get("model_id") is not None,
+    )
+
+    try:
+        task_dict = task_kinds_service.create_task_or_append(
+            db, obj_in=task_create, user=current_user, task_id=task_id
+        )
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)